File: PRACTICA/Java/irrgarten/PlayerSquare.java
package irrgarten;

public class PlayerSquare {
    private int row;
    private int col;
    private Player player;

    public PlayerSquare(int row, int col, Player player) {
        this.row = row;
        this.col = col;
        this.player = player;
    }

    public Player get() {
        return player;
    }

    public void set(int row, int col, Player player) {
        this.row = row;
        this.col = col;
        this.player = player;
    }

    public String toString() {
        return player.getNumber() + "";
    }
}

File: PRACTICA/Ruby/Irrgarten/player_square.rb
class Player_square
    def initialize(row, col, name)
        @row = row
        @col = col
        @name = name
    end

    def get()
        return @name
    end

    def set(row, col, name)
        @row = row
        @col = col
        @name = name
    end
    
    def to_s()
        return @name.get_number() + ""
    end
end